docs/source/tutorials_source/package/tutorial_moco_memory_bank.py

```diff
@@ -324,6 +333,7 @@ def configure_optimizers(self):
 
 # %%
 # Train the Classifier
+model.eval()
 classifier = Classifier(model.resnet_moco)
 trainer = pl.Trainer(max_epochs=max_epochs, gpus=gpus,
                      progress_bar_refresh_rate=100)
```

Comment:
Did you verify that the results work after changing the learning rate, batch size etc?

Reply:
Yes, accuracy is higher, model trains faster due to larger batch size. However, I also added cosine LR and therefore increased the default number of epochs. I will update the exact numbers in the docs. So far I got around 80% after 200 epochs.